hazelcast/src/main/java/com/hazelcast/partition/impl/PromoteFromBackupOperation.java

```diff
@@ -64,6 +66,42 @@ public void run() throws Exception {
         }
     }
 
+    private void setMissingReplicaVersions() {
```

Comment:
Thanks to @metanet, this is inspired by his commit: https://github.com/metanet/hazelcast/commit/7a1f882c961bb56c7f10c6d20a5e2acf446944e4